FlagsmithClient/src/main/java/com/flagsmith/Flagsmith.kt

```diff
@@ -101,6 +165,20 @@ class Flagsmith constructor(
             analytics?.trackEvent(featureId)
             foundFlag
         })
+    }.also { lastUsedIdentity = identity }
+
+    private fun getFlagUpdates() {
```

Comment:
Shouldn't this be named `startRealTimeListener` or something?

Reply:
Done